packages/react-components/react-button/library/src/components/CompoundButton/useCompoundButtonStyles.styles.ts

```diff
@@ -132,37 +133,37 @@ const useRootStyles = makeStyles({
   small: {
     padding: `${tokens.spacingHorizontalS} ${tokens.spacingHorizontalS} ${tokens.spacingHorizontalMNudge} ${tokens.spacingHorizontalS}`,
 
-    fontSize: tokens.fontSizeBase300,
-    lineHeight: tokens.lineHeightBase300,
+    fontSize: semanticTokens.textRampItembodyFontsize,
+    lineHeight: semanticTokens.textRampItembodyLineheight,
   },
   medium: {
     padding: `14px ${tokens.spacingHorizontalM} ${tokens.spacingHorizontalL} ${tokens.spacingHorizontalM}`,
 
-    fontSize: tokens.fontSizeBase300,
-    lineHeight: tokens.lineHeightBase300,
+    fontSize: semanticTokens.textRampItembodyFontsize,
```

Comment:
Is the word case correct in here? "Itembody" vs "ItemBody"

Reply:
The token names are standardized and enforced by the centralized definitions so there were correct, however they've since done a camel casing parse and ensured everything is aligned now so these should be camel cased as expected after rebasing.